source/grapher.js

```diff
@@ -811,24 +831,7 @@ grapher.Edge = class {
         edgePathGroupElement.appendChild(this.element);
         this.hitTest = createElement('path');
         this.hitTest.setAttribute('class', 'edge-path-hit-test');
-        if (this.focus) {
-            this.hitTest.addEventListener('pointerover', (e) => {
-                this.focus();
-                e.stopPropagation();
-            });
-        }
-        if (this.blur) {
-            this.hitTest.addEventListener('pointerleave', (e) => {
-                this.blur();
-                e.stopPropagation();
-            });
-        }
-        if (this.activate) {
-            this.hitTest.addEventListener('click', (e) => {
-                this.activate();
-                e.stopPropagation();
-            });
-        }
+        this.hitTest._var = this;
```

Comment:
Can this be accomplished without storing objects on HTML elements?

Reply:
I've updated the code. 2090f75554b5409170703ef28d5fe06016a1e59a